app/models/reports/repository.rb

```diff
@@ -149,14 +147,12 @@ def self.smart_memoize(method)
       end
     end
 
-    # If we are calculating percentages when with_exclusions is true, we have to manually subtract out the LTFU
-    # patient counts as well from the patient counts.
+    # This method currently always returns the "excluding LTFU denominator".
+    # Repository only returns "excluding LTFU" rates.
+    # This only powers queries for children regions which do not require both variants of control rates, unlike Result.
+    # As we deprecate Result and shift to Repository, a Repository object should be able to return both rates.
     def denominator(region, period)
-      if with_exclusions
-        cumulative_assigned_patients_count[region.slug][period.adjusted_period] - ltfu_counts[region.slug][period]
-      else
-        cumulative_assigned_patients_count[region.slug][period.adjusted_period]
-      end
+      cumulative_assigned_patients_count[region.slug][period.adjusted_period] - ltfu_counts[region.slug][period]
```

Comment:
@rsanheim This method now only return the `with_exclusions` version right now, since we don't use the included version anywhere. When we decide to pull rates from this class, we'll need to do something like https://github.com/simpledotorg/simple-server/blob/8949d9ef94765ef0813f25518ac1f3f4e21c0c58/app/models/reports/result.rb#L218